projects/packages/image-cdn/src/class-image-cdn.php

```diff
@@ -399,11 +399,11 @@ public static function filter_the_content( $content ) {
 					// First, check the image tag. Note we only check for pixel sizes now; HTML4 percentages have never been correctly
 					// supported, so we stopped pretending to support them in JP 9.1.0.
 					if ( preg_match( '#[\s"\']width=["\']?([\d%]+)["\']?#i', $images['img_tag'][ $index ], $width_string ) ) {
-						$width = false === strpos( $width_string[1], '%' ) ? $width_string[1] : false;
+						$width = ! str_contains( $width_string[1], '%' ) ? $width_string[1] : false;
 					}
 
 					if ( preg_match( '#[\s"\']height=["\']?([\d%]+)["\']?#i', $images['img_tag'][ $index ], $height_string ) ) {
-						$height = false === strpos( $height_string[1], '%' ) ? $height_string[1] : false;
+						$height = ! str_contains( $height_string[1], '%' ) ? $height_string[1] : false;
```

Comment:
(same, just leaving the suggestion here too)
```suggestion
						$height = str_contains( $height_string[1], '%' ) ? false : $height_string[1];
```